api/.eslintignore

```diff
@@ -9,4 +9,6 @@ jspm_packages
 .webpack
 
 src/setupTests.js
-src/teardownTests.js
\ No newline at end of file
+src/teardownTests.js
+# Large file with binary/hex contents
+api/wiremock/__files/tax-clearance-success.json
```

Comment:
this exist in the root file, I added it here, prettier hang when it's checking this file, it makes sense to add it here